<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>params_config2</class>
 <widget class="QWidget" name="params_config2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>int</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>double</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数个数：2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="range_max">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="range_min">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="next_step">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QLabel" name="Kind">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数1类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="Range">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数1范围:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QPushButton" name="former_step">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="range_max_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="range_min_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Kind_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数2类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>int</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>double</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Range_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数2范围:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
